src/commands/transaction/create/transfer.js

```diff
@@ -19,10 +19,22 @@ import BaseCommand from '../../../base';
 import commonFlags from '../../../utils/flags';
 import getInputsFromSources from '../../../utils/input';
 
-const processInputs = (amount, address) => ({ passphrase, secondPassphrase }) =>
+const dataFlag = {
+	char: 'd',
+	description: `Specifies the data for the transaction type 0. Takes a string.
```

Comment:
I think description can be improved.
`Message of  UTF-8 string to attach, which has a maximum length of 64 bytes`
@mitsujutsu what do you think?

Reply:
I agree, I checked on Lisk Documentation and its the same so modified it